openqa-investigate

```diff
@@ -51,13 +51,24 @@ clone() {
     base_prio=$(echo "$clone_job_data" | runjq -r '.job.priority') || return $?
     clone_settings=("TEST+=:investigate$name_suffix" '_TRIGGER_JOB_DONE_HOOK=1' '_GROUP_ID=0' 'BUILD=')
     if [[ $refspec ]]; then
-	vars_json=$(fetch-vars-json "$origin")
+        vars_json=$(fetch-vars-json "$origin")
         testgiturl=$(echo "$vars_json" | runjq -r '.TEST_GIT_URL')
         casedir=$(echo "$clone_job_data" | runjq -r '.job.settings.CASEDIR') || return $?
         [[ $testgiturl != null ]] && casedir=$testgiturl
         [[ $casedir == null ]] && casedir=''
         repo=${casedir:-'https://github.com/os-autoinst/os-autoinst-distri-opensuse.git'}
         clone_settings+=("CASEDIR=${repo%#*}#${refspec}")
+        worker_class=$(echo "$vars_json" | runjq -r '.WORKER_CLASS') || return $?
+        used_worker=$(echo "$clone_job_data" | runjq -r '.job.settings.WORKER_CLASS') || return $?
```

Comment:
The variable names seem to me as if they should be swapped.
The actual *used* worker is the one in the vars.json, the requested worker class is in the job settings, right?